src/transformers/utils/sentencepiece_model_pb2.py

```diff
@@ -0,0 +1,1169 @@
+# flake8: noqa
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
Do we need to have this in the utils?

Reply:
Yes, we have to have this file for now.
Could in another place though, I don't have a strong opinion.